src/internal/connector/onedrive/collection.go

```diff
@@ -347,6 +368,41 @@ func downloadContent(
 	return content, nil
 }
 
+// readFromCache fetches latest download URL from the cache and attempts to
+// download the file using the new URL.
+func (oc *Collection) readFromCache(
```

Comment:
`cache` can take on many terms, and it's possible we'll have multiple caches.  Best practice is to denote what type of cache you're reading as part of the func name.  Eg: `readFromURLCache`.